titles/getting-started-rhdh/title-getting-started.adoc

```diff
@@ -10,9 +10,6 @@ As a developer, you can use {product} to experience a streamlined development en
 //customer support links
 include::artifacts/snip-customer-support-info.adoc[]
 
-// rhdh overview
-include::modules/installation/con-rhdh-overview.adoc[leveloffset=+1]
```

Comment:
This pull request should contain onlyu changes to thos file and the titles/rel-notes-rhdh/title-rhdh-release-notes.adoc file